account_statement_base/views/account_bank_statement_line.xml

```diff
@@ -145,9 +145,9 @@
                 />
                 <separator />
                 <filter
-                    name="to_check"
+                    name="checked"
                     string="To check"
-                    domain="[('to_check', '=', True)]"
+                    domain="[('checked', '=', True)]"
                 />
```

Comment:
Why not keeping this filter ?
```suggestion
                <filter
                    name="to_check"
                    string="To check"
                    domain="[('checked', '=', False)]"
                />
```

Reply:
> checked was replace to_checked fields in commit: https://github.com/odoo/odoo/commit/242e1a04fc97131ccc28905de58011127f078e30

Reply:
hi @mmequignon,
I have keeped it. thanks